### PR TITLE
Add new pager display mode

### PR DESCRIPTION
**Pull request by ghsteff**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

There's no options to display other types of pagers

## What is the new behavior?
This adds a new pager prop `displayMode` to change the type of pager that's rendered

also adds `pagesContainerClassName` and `activePageClassName` to for more flexible styling customization

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information
image

### CODE REVIEW

File: src/data/Pager/Pager.tsx [hunk 1]
```diff
@@ -60,24 +72,33 @@ export interface PagerProps {
    * A callback function that is called when the page changes.
    */
   onPageChange?: (page: number) => void;
+
+  /**
+   * The type of table data for the pager to display.
+   */
+  displayMode?: 'pages' | 'items' | 'all';
```
Comment (amcdnl) [inline src/data/Pager/Pager.tsx:79]: The `items` name isn't the most intuitive to me - what about `controls`?

- `displayMode: 'all' | 'controls' | 'pages';`

File: src/data/Pager/Pager.module.css [hunk 2]
```diff
@@ -16,3 +11,18 @@
     }
   }
 }
+
+.pagesContainer {
+  display: flex;
+  align-items: center;
+}
+
+svg {
```
Comment (amcdnl) [inline src/data/Pager/Pager.module.css:20]: This is a global style - we need to scope it at least to a parent.

**Review by amcdnl — CHANGES_REQUESTED** [review]
minor items

File: src/data/Pager/Pager.tsx (hunk 1 above)
Comment (ghsteff) [reply src/data/Pager/Pager.tsx:79]: the `items` option is controlling the display for the `1-10 of 100 items` part not the arrow buttons. Maybe `label` or `rows` could be a diff name?

**Review by amcdnl — APPROVED** [review]

**Pull request merged**